package.json

```
@@ -92,7 +92,7 @@
 		"build:mac": "yarn run clean && electron-builder --mac dmg:x64",
 		"build:mac:arm64": "yarn run clean && electron-builder --mac dmg:arm64",
 		"build:win": "yarn run clean && electron-builder --win",
-		"lint": "xo",
+		"lint": "rome ci .",
```

Should we keep lint for fixing and have a separate command for checking?
```suggestion
		"lint": "rome format",
		"lint:check": "rome ci",
```
and then have CI call `yarn lint:check .`

---

fixed by https://github.com/th-ch/youtube-music/pull/1062/commits/fa069d11b8faf5758a71f8a3d94216726c0fbe11